src/Map/TileMap.h

```diff
@@ -79,7 +79,7 @@ class TileMap: public micropather::Graph
 	/** MicroPather public interface implementation. */
 	virtual float LeastCostEstimate(void* stateStart, void* stateEnd);
 	virtual void AdjacentCost(void* state, std::vector<micropather::StateCost>* adjacent);
-	virtual void PrintStateInfo(void* state) {};
+	virtual void PrintStateInfo(void* /*state*/) { /* DO NOTHING */ };
```

Comment:
I would prefer to not include `/* DO NOTHING */` comments. It doesn't really tell us anything that the empty `{}` doesn't. The best comment is no comment.

Same for all the other similar comments.

Reply:
Sorry about that. It's a (bad?) habit to let people and future me know that the implementation isn't supposed to do anything on purpose. Fixed now.